packages/devtools_app/lib/src/screens/debugger/program_explorer_controller.dart

```diff
@@ -352,12 +352,16 @@ class ProgramExplorerController extends DisposableController
     final LibraryRef targetLib = scriptObj.library!;
 
     // Search targetLib only on the root level nodes.
-    final libNode = _searchRootObjectNodes(targetLib)!;
+    var libNode = _searchRootObjectNodes(targetLib);
+    libNode ??= breadthFirstSearchObject(
```

Comment:
can you add a comment here explaining in which case the libNode would be null and then we'd have to run the bfs?